homestar-runtime/src/network/webserver.rs

```diff
@@ -1,4 +1,4 @@
-//! Sets up a websocket server for sending and receiving messages from browser
+//! Sets up a webserver for websocket and HTTP interaction with external
```

Comment:
```suggestion
//! Sets up a webserver for websocket and HTTP interaction with
```

Possible edit. All clients are external.